multigeiger/display.cpp

```diff
@@ -35,18 +35,16 @@ void display_start_screen(void) {
   }
 
   if (isLoraBoard) {
-    pu8x8->drawString(0, 2, "Geiger-");
-    pu8x8->drawString(0, 3, " Counter");
-    pu8x8->drawString(0, 4, "Version:");
+    pu8x8->drawString(0, 2, "MGeiger");
+    pu8x8->drawString(0, 3, "Version:");
     snprintf(line, 9, "%s", VERSION_STR);  // 8 chars + \0 termination
-    pu8x8->drawString(0, 5, line);
+    pu8x8->drawString(0, 4, line);
   } else {
     pu8x8->setFont(u8x8_font_amstrad_cpc_extended_f);
-    pu8x8->drawString(0, 0, "Geiger-Counter");
-    pu8x8->drawString(0, 2, "==============");
+    pu8x8->drawString(0, 0, "Multi-Geiger");
+    pu8x8->drawString(0, 2, "Info:boehri.de");
     snprintf(line, 15, "%s", VERSION_STR);  // 14 chars + \0 termination
     pu8x8->drawString(0, 4, line);
-    pu8x8->drawString(0, 6, "Info:boehri.de");
```

Comment:
changes are still in here?

Reply:
some changes are required for the PR to run, will remove, rebase and squash as soon as the other PR is through.